sygnal/fcmpushkin.py

```diff
@@ -37,38 +37,38 @@
 from .notifications import ConcurrencyLimitedPushkin
 
 QUEUE_TIME_HISTOGRAM = Histogram(
-    "sygnal_gcm_queue_time", "Time taken waiting for a connection to GCM"
+    "sygnal_fcm_queue_time", "Time taken waiting for a connection to FCM"
 )
 
 SEND_TIME_HISTOGRAM = Histogram(
-    "sygnal_gcm_request_time", "Time taken to send HTTP request to GCM"
+    "sygnal_fcm_request_time", "Time taken to send HTTP request to FCM"
 )
 
 PENDING_REQUESTS_GAUGE = Gauge(
-    "sygnal_pending_gcm_requests", "Number of GCM requests waiting for a connection"
+    "sygnal_pending_fcm_requests", "Number of FCM requests waiting for a connection"
 )
 
 ACTIVE_REQUESTS_GAUGE = Gauge(
-    "sygnal_active_gcm_requests", "Number of GCM requests in flight"
+    "sygnal_active_fcm_requests", "Number of FCM requests in flight"
 )
 
 RESPONSE_STATUS_CODES_COUNTER = Counter(
-    "sygnal_gcm_status_codes",
-    "Number of HTTP response status codes received from GCM",
+    "sygnal_fcm_status_codes",
+    "Number of HTTP response status codes received from FCM",
     labelnames=["pushkin", "code"],
 )
 
 logger = logging.getLogger(__name__)
 
-GCM_URL = b"https://fcm.googleapis.com/fcm/send"
+FCM_URL = b"https://fcm.googleapis.com/fcm/send"
 MAX_TRIES = 3
 RETRY_DELAY_BASE = 10
 MAX_BYTES_PER_FIELD = 1024
 
 # The error codes that mean a registration ID will never
 # succeed and we should reject it upstream.
 # We include NotRegistered here too for good measure, even
-# though gcm-client 'helpfully' extracts these into a separate
+# though fcm-client 'helpfully' extracts these into a separate
 # list.
 BAD_PUSHKEY_FAILURE_CODES = [
```

Comment:
```suggestion
BAD_PUSHKEY_FAILURE_CODES = [
```

`gcm-client` was a library for using GCM, so renaming the reference to fcm-client doesn't make sense anymore.

However, gcm-client isn't even in use, so this comment is probably _very_ crufty — hence I think it should just be removed altogether.